en/docs/troubleshoot/faq.md

```diff
@@ -0,0 +1,147 @@
+- **Cloud Native App Developer**: Design, develop, deploy, and operate your backend APIs, services, integrations, and more with any programming language.
+- **DevOps**: Release and manage applications on Kubernetes without compromising on security, reliability, or governance, across any cloud.
+
+### Q: What is an organization in Choreo?
+An organization is a logical grouping of users and their resources. It may represent a company, community, or a single user. Users can belong to multiple organizations, and each organization can have different roles assigned to its users to control access to Choreo features.
+
+### Q: What is a project in Choreo?
+A project is a logical grouping of related components to help you to organize your work. Each project provides runtime isolation through namespaces when you deploy components.
+
+### Q: What is a component in Choreo?
+A component is a workload designed to run on Choreo. Examples of components include integrations, APIs, microservices, manual/scheduled jobs, web apps, and triggers.
+
+### Q: What is the difference between an internal and external API?
+ In Choreo, you can publish an API as an internal or an external API.  A user or an application can access an external API publicly over the internet whereas an internal API is only accessible through other components within the same organization. 
+
+### Q: What is a connector in Choreo Marketplace?
+A connector is a reusable Ballerina package that simplifies connecting to external or internal systems and APIs, such as Salesforce, SAP, GitHub, and Twilio. You can use the connectors available in the Choreo marketplace to implement your integration use cases.  Connectors can be created and published by both WSO2 and Choreo users.
+
+### Q: What is a trigger in Choreo Marketplace?
+A trigger is a construct that enables users to receive known event payloads from external systems, facilitating event-driven programming.
+
+### Q: What is a sample/template in Choreo?
+A sample or template is a prebuilt Ballerina program that covers a popular integration use case or pattern. Examples include connecting Salesforce to Slack or implementing content-based routing.
+
+### Q: What are the support options in Choreo?
+You can find information about our support plans, including `free`, `basic`, and `enterprise` options at [https://wso2.com/choreo/customer-support/](https://wso2.com/choreo/customer-support/).
+
+### Q: How can I perform log monitoring or analytics for the Azure environment?
+If you have a log monitoring product or service, such as Azure Monitor, you can use it together with Choreo. Note: The log monitoring tool is not included in the infrastructure cost.
+
+### Q: What source control software does Choreo support?
+Choreo currently supports GitHub. Support for Bitbucket and GitLab is on our roadmap. 
```

Comment:
Now we support Bitbucket too. So I suppose we would need to rephrase accordingly.

Reply:
changed